tests/cupy_tests/manipulation_tests/test_basic.py

```diff
@@ -125,3 +126,47 @@ def test_copyto_where(self, xp, dtype):
             self.dst_shape, xp, dtype) % 2).astype(xp.bool_)
         xp.copyto(dst, self.src, where=mask)
         return dst
+
+
+@testing.parameterize(
+    *testing.product(
+        {'shape': [(0,), (1,), (2, 3), (2, 3, 4)]}))
+@testing.gpu
+class TestPutmask(unittest.TestCase):
+
+    @testing.for_all_dtypes()
+    @testing.numpy_cupy_allclose()
+    def test_putmask(self, xp, dtype):
+        a = testing.shaped_arange(self.shape, xp, dtype=dtype)
+        mask = testing.shaped_random(self.shape, xp, dtype=bool)
+        values = testing.shaped_random(self.shape, xp, dtype=dtype)
```

Comment:
Feed different seeds.
```suggestion
        a = testing.shaped_arange(self.shape, xp, dtype=dtype, seed=0)
        mask = testing.shaped_random(self.shape, xp, dtype=bool, seed=1)
        values = testing.shaped_random(self.shape, xp, dtype=dtype, seed=2)
```